app/views/episodes/_form_status.html.erb

```diff
@@ -41,9 +41,8 @@
 
       <% if policy(episode).update? %>
         <div>
-          <%= link_to t(".discard"), edit_episode_path(episode), class: "btn btn-discard-changed invisible", data: {action: "unsaved#discard", unsaved_target: "discard"} %>
-
           <div class="btn-group">
+            <%= link_to t(".discard"), edit_episode_path(episode), class: "btn btn-discard-changed d-none me-2", data: {action: "unsaved#discard", unsaved_target: "discard"} %>
```

Comment:
If the discard button is within the "btn-group" it helps a lot of the default spacing, ensures the buttons remain inline.

Reply:
This does ruin the rounded corners on the left side of the save button?

<img src="https://github.com/PRX/feeder.prx.org/assets/1410587/056b434c-25f0-4832-a2a7-215eeabafba0" width=400/>